google/cloud/spanner/admin/integration_tests/instance_admin_rest_integration_test.cc

```diff
@@ -0,0 +1,430 @@
+// Copyright 2023 Google LLC
```

Comment:
Are we concerned about all the duplication between REST and non-REST hand-written code?

Reply:
Yes, I think we should change the existing integration tests if at all possible.

Reply:
I have reduced the amount of duplication such that there are tests sufficiency to verify that REST transport works a) with the emulator and b) with production, and we test LRO rpcs, paginated rpcs, and vanilla rpcs.